manifests/unzip.pp

```diff
@@ -50,23 +50,35 @@
   $refreshonly      = false,
   $unless           = undef,
   $zipfile          = $name,
-  $provider         = 'powershell',
+  $provider         = 'dotnet',
   $options          = '20',
-  $command_template = 'windows/unzip.ps1.erb',
   $timeout          = 300,
+  $logoutput        = on_failure,
 ) {
   validate_absolute_path($destination)
 
   if (! $creates and ! $refreshonly and ! $unless){
     fail("Must set one of creates, refreshonly, or unless parameters.\n")
   }
 
+  unless( $provider == 'dotnet'  or $provider == 'com'){
```

Comment:
What happened to the old default of `powershell`? Should that be supported for backwards compatibility?

Also, some funky spacing in this line.